beancount_n26/__init__.py

```diff
@@ -85,6 +99,27 @@ def name(self):
     def file_account(self, _):
         return self.account
 
+    def file_date(self, file_):
+
+        if not self.identify(file_):
+            return None
+
+        date = None
+
+        with open(file_.name, encoding=self.file_encoding) as fd:
+            reader = csv.DictReader(
+                fd, delimiter=',', quoting=csv.QUOTE_MINIMAL, quotechar='"'
+            )
+
+            for line in reader:
+                date_tmp = datetime.strptime(
+                    line[self._translate('date')], '%Y-%m-%d'
+                )
+                if not date or date_tmp > date:
+                    date = date_tmp
+
+        return date.date()
```

Comment:
One last change request. For empty files, `date` would be `None` here so `date.date()` would not work. Could you replace this with `return date.date() if date else None`?

Reply:
Sure, it is sorted.